### PR TITLE
Safari click area fix

### LINKED ISSUE #72: [Safari] mobile hit area shifting
### Terms

- [X] I have searched all [open bug reports](https://github.com/activist-org/activist/issues?q=is%3Aopen+is%3Aissue+label%3Abug)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Behavior

Safari is currently having issues with the landing page on mobile. Specifically some of the page elements’ hit areas are not coinciding with their positioning on the page. `LabeldBtns` at times aren’t clickable within their boundaries, and the elements of the supporters section at times have hit areas that extend beyond their boundaries and interfere with other elements on the page being accessible.

### PR DESCRIPTION
**Pull request by anthonyshibitov**
Fix Safari mobile click area bug by explicitly stating the height and width of the Logo components in LandingPageLogoImage. 

### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch
  
  

---

### Description

Previously, the clickable area was spilling out of the LandingPageLogoImage component.

Since the SVGs are now resized to this 200/250px width, scale-[x] can be mostly removed except to fix small spacing/aesthetic issues. I left scaling in the WikimediaRS and ImpactHub components to match what it looked like beforehand.

This could be further fixed by using SVGs specifically made for the Supporters section, instead of scaling them via tailwind.

---

I've confirmed that this is now working on my iPhone, and the click area is no longer spilling from the logo area.

Note, that I think this is an issue with tailwind, as this isn't reproducible on Safari emulating an iPhone. 

### Related issue

 - Fixes #72

### CODE REVIEW

**Comment by andrewtavis** [comment]
Hey @anthonyshibitov! In checking it out this is firstly working really well, but I realized the big thing that's still causing a problem :) Maybe you can check to see if you can also reproduce on your end with the commits in the PR, and if so we can think about how to fix that 😊 

I'm realizing that some of the buttons are not working still, but that's specifically because when you try to click them we're instead clicking the logo in the header (try long holding them and you see that the preview is the landing page itself). The `Request access` button right at the top works for me if I scroll down a bit such that the header disappears, but then if the header's there then I just click that instead and even see the logo blink a bit.

Could you let me know if you're also experiencing this?

**Comment by anthonyshibitov** [comment]
Ahhh, okay.

The header logo button is extending down to around the center of the page for some reason. Is this what you're talking about? Sometimes clicking a button when its underneath this header click-area it will click the header logo instead and make it grey, and not register the actual button click.

Here's a screenshot of it mid load, showing the actual clickable area. In this example I tried clicking the `Request access` button, but it registered clicking the `activist` logo at the top.

IMG_9063

I didn't realize this was an issue, I was just looking at the supporter logos at first. I'll dig into this!

**Comment by andrewtavis** [comment]
> I didn't realize this was an issue, I was just looking at the supporter logos at first. I'll dig into this!

Ya I didn't think of it until I realized that it wasn't happening when the header was hidden :) Thank you! 🙌

**Comment by andrewtavis** [comment]
I'm going to bed now 😅 I'd try adding `overflow-clip` to the logo's class if you haven't already :) This tends to restrict the bounds appropriately 🤞

**Comment by andrewtavis** [comment]
If not to the logo, than to the header's class :)

**Comment by anthonyshibitov** [comment]
I believe it already has `overflow-clip`, no?

https://github.com/activist-org/activist/blob/main/frontend/components/WebsiteHeader.vue#L16

**Comment by anthonyshibitov** [comment]
Alrighty, check out this commit and let me know what you think.

I tried adding `overflow-clip` to a few different elements and couldn't get the desired behavior.

Let me know if `overflow-clip` works instead somewhere else - I think that's probably a better way to fix this than explicitly stating the dimension the SVG _should_ be. I'll revert this commit and commit the `overflow-clip` change if that's preferred, just let me know! :)

It's weird though. Even though in the Chrome/Mac Safari element inspector it shows that the SVG is expanded beyond it's parents container, it still doesn't register it as a click - only on mobile Safari.

Before
Screenshot 2023-05-23 at 8 53 45 PM

After
Screenshot 2023-05-23 at 8 53 59 PM

**Comment by andrewtavis** [comment]
Let's definitely ignore my `overflow-clip` suggestion :) This is really great! The one thing is that we'd be applying the size to the logo file itself rather than to the instance in the mobile header, so it's thus shrinking it to being too small on desktop. Maybe applying the style [directly on the mobile header](https://github.com/activist-org/activist/blob/71eff59534cd1f088f160bfd705dc07f28ec557a/frontend/components/WebsiteHeader.vue#L15)? We're also getting rid of some of these component SVGs soon, btw. I historically had a hard time getting them to change color with the system color scheme, but there are better ways of doing this than having them be components.

**Comment by anthonyshibitov** [comment]
So unfortunately the only way I found this was able to work was by changing the SVG class directly. 

I’ve tried passing a `isMobile` prop to the logo component and this seems to work, but it might be a bit verbose. Let me know if there’s a better way of doing this. Thanks!!

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (andrewtavis) [inline yarn.lock:1]: Looks like you yarn installed in the root :) I'll go ahead and merge and delete the file 🙃

**Comment by andrewtavis** [comment]
Checked with that friend about whether you can pick up #121 or #122, btw. Will let you know and then you'd just need to write in the one that we decide on so that I can assign you 😊

I have some local changes for that file that I'll send later on tonight with a bunch of updates. Should be good to start tomorrow :)

**Comment by anthonyshibitov** [comment]
Sounds good!

**Pull request merged**